File: 3_control_flow/control_flow.c
#include <stdio.h>
#include <stdlib.h>
int main(int argc, char **argv)
{
	int i=atoi(argv[1]);
	int j=atoi(argv[2]);

	if(i==j)
		printf("i==j\n");
	else
		printf("i!=j\n");

	/* error example */
	if(i==j)
		printf("i==j\n");
	else
		printf("enter else condition\n");
		printf("i!=j\n");
		
	printf("======= while ===========\n");
	while(i<100)
	{
		printf("i:%d\n",i);
		i++;
	}

	printf("======= for ===========\n");
	for(i=100;i>=0;i--)
		printf("i:%d\n",i);

	exit(EXIT_SUCCESS);
}

File: 3_control_flow/control_flow.py
import sys

def main():
    i=int(sys.argv[1])
    j=int(sys.argv[2])

    if(i==j):
        print("i==j\n")
    else:
        print("i!=j\n")

    #python does not like c
    if(i==j):
        print("i==j\n")
    else:
        print("enter else condition\n")
        print("i!=j\n")

    print("======= while ===========\n")
    while(i<100):
        print("i:",i)
        i=i+1	

    print("======= for ===========\n")
    i=0
    for i in range(100):
        print("i:",100-i)


if __name__ == "__main__":
    main()
